From 42088d25bd3e44a6c2ccfaa5ef4312be5b8eba03 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Tue, 16 Jun 2020 10:24:34 -0500
Subject: [PATCH 3/6] configs: beacon-rzg2m: Correct Copyrights to reflect
 Compass

The parent company of Beacon EmbeddedWorks is Compass Electronics
Group, LLC.  This patch changes the name of the copyright holder
and eliminates dead code.  No functional changes are present.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 include/configs/beacon-rzg2m.h | 13 ++++---------
 1 file changed, 4 insertions(+), 9 deletions(-)

diff --git a/include/configs/beacon-rzg2m.h b/include/configs/beacon-rzg2m.h
index b29f0a9215..f5b229ec2b 100644
--- a/include/configs/beacon-rzg2m.h
+++ b/include/configs/beacon-rzg2m.h
@@ -1,21 +1,16 @@
 /* SPDX-License-Identifier: GPL-2.0+ */
 /*
- * include/configs/hihope-rzg2m.h
- *     This file is HOPERUN HiHope-RZG2M board configuration.
- *
- * Copyright (C) 2019 Renesas Electronics Corporation
+ * Copyright (C) 2020 Compass Electronics Group, LLC
  */
 
-#ifndef __HIHOPE_RZG2M_H
-#define __HIHOPE_RZG2M_H
+#ifndef __BEACON_RZG2M_H
+#define __BEACON_RZG2M_H
 
 #include "rcar-gen3-common.h"
 
 /* Ethernet RAVB */
-#if 1
 #define CONFIG_BITBANGMII
 #define CONFIG_BITBANGMII_MULTI
-#endif
 
 /* Generic Timer Definitions (use in assembler source) */
 #define COUNTER_FREQUENCY	0xFE502A	/* 16.66MHz from CPclk */
@@ -25,4 +20,4 @@
 #define CONFIG_SYS_MMC_ENV_DEV		1
 #define CONFIG_SYS_MMC_ENV_PART		2
 
-#endif /* __HIHOPE_RZG2M_H */
+#endif /* __BEACON_RZG2M_H */
-- 
2.17.1

